resources/todomvc/architecture-examples/angular/src/styles.css

```diff
@@ -22,3 +22,12 @@ but useful for people who use screen readers */
 .toggle-all-label {
     pointer-events: none;
 }
+
+app-todo-item {
+    display: block;
+    border-bottom: 1px solid #ededed;
+}
+
+app-todo-item:last-child {
+    border-bottom: none;
+}
```

Comment:
Interestingly, :last-child can be costly, and this might explain why all browsers see a slight increase in their runtime. But I don't mind keeping it as it's very likely that this is used this way in real-world websites.